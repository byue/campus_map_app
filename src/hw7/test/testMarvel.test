# Tests various inputs for the marvel graph. Expected outputs were found by
# browsing the marvel.tsv data and tallying counts of books with a dictionary.

LoadGraph marvel marvel.tsv

FindPath marvel HOFFMAN CAPTAIN_AMERICA

FindPath marvel W'KABI LYNNE,_MONICA

FindPath marvel CAPTAIN_AMERICA HAWK

FindPath marvel HAWK HAWK

FindPath marvel HAWK @#$