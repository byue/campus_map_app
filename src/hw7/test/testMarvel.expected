# Tests various inputs for the marvel graph. Expected outputs were found by
# browsing the marvel.tsv data and tallying counts of books with a dictionary.

loaded graph marvel

path from HOFFMAN to CAPTAIN AMERICA:
no path found

path from W'KABI to LYNNE, MONICA:
W'KABI to BLACK PANTHER/T'CHAL with weight 0.091
BLACK PANTHER/T'CHAL to LYNNE, MONICA with weight 0.021
total cost: 0.112

path from CAPTAIN AMERICA to HAWK:
CAPTAIN AMERICA to HAWK with weight 0.003
total cost: 0.003

path from HAWK to HAWK:
total cost: 0.000

unknown character @#$